Realm.xcodeproj/project.pbxproj

```diff
@@ -1235,6 +1277,20 @@
 			shellPath = /bin/sh;
 			shellScript = "echo \"#define REALM_COCOA_VERSION @\\\"$(sh build.sh get-version)\\\"\" > ${DERIVED_FILE_DIR}/RLMVersion.h";
 		};
+		C04B4BDB1B55C47A00FAE58E /* Switch Swift Version */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "Switch Swift Version";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if [[ (-L Realm/Tests/Swift) || (! -d Realm/Tests/Swift) ]]; then\n    version=\"$(xcrun swift --version | grep -io --color=never -E 'Apple Swift version ([^\\b ]*)' | cut -f 4 -d ' ')\"\n    rm -f Realm/Tests/Swift\n    (cd Realm/Tests && ln -s \"Swift$version\" Swift)\nfi";
```

Comment:
You can extract the Swift version with `sed` (or `perl`) rather than combining `grep` and `cut`:

    sed -ne 's/^Apple Swift version \([^\b ]*\).*/\1/p'
    perl -nle '/Apple Swift version ([^\b ]*)/ && print $1'

To me it's also a little more obvious what they're doing.